fhir-install/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN unzip -qq /tmp/fhir-server-distribution.zip -d /tmp && \
 FROM openliberty/open-liberty:20.0.0.9-full-java8-openj9-ubi
 COPY --chown=1001:0 --from=base /opt/ol/wlp /opt/ol/wlp
 
-MAINTAINER Lee Surprenant <lmsurpre@us.ibm.com>
+LABEL MAINTAINER="Lee Surprenant <lmsurpre@us.ibm.com>"
```

Comment:
I'd be fine to totally drop this line...similar to how we decied now to include the `@author` tag on the source files

Reply:
OK - will remove